atc/exec/get_step.go

```diff
@@ -268,3 +299,51 @@ func (step *GetStep) run(ctx context.Context, state RunState, delegate GetDelega
 
 	return succeeded, nil
 }
+
+func (step *GetStep) getFromLocalCache(
+	logger lager.Logger,
+	teamId int,
+	resourceCache db.UsedResourceCache,
+	workerSpec worker.WorkerSpec) (worker.GetResult, bool, error) {
+	volume, found := step.findResourceCache(logger, teamId, resourceCache, workerSpec)
+	if !found {
+		return worker.GetResult{}, false, nil
+	}
+	metadata, err := resourceCache.LoadVersionMetadata()
```

Comment:
@aoldershaw pointed out that we can actually use an existing method `ResourceCacheMetadata` rather than adding a new method here! https://github.com/concourse/concourse/blob/ff1d2fae5eb0e657987ec3af3f1162f5d63db698/atc/db/resource_cache_factory.go#L103

Reply:
Fixed.